src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -106,17 +106,17 @@ void OnSystemRequestNotification::Run() {
     }
   }
 
+  const std::string file_name =
+      (*message_)[strings::msg_params][strings::file_name].asString();
+  BinaryMessage binary_data;
   if (mobile_apis::RequestType::PROPRIETARY == request_type) {
     /* According to requirements:
        "If the requestType = PROPRIETARY, add to mobile API fileType = JSON
         If the requestType = HTTP, add to mobile API fileType = BINARY"
        Also in Genivi SDL we don't save the PT to file - we put it directly in
        binary_data */
-
-    const std::string filename =
-        (*message_)[strings::msg_params][strings::file_name].asString();
-    BinaryMessage binary_data;
-    file_system::ReadBinaryFile(filename, binary_data);
+    LOG4CXX_DEBUG(logger_, "According to requirements");
```

Comment:
@ZhdanovP Unclear log...Remove it please

Reply:
@AByzhynar please take a look f2dd4a6562711fbaaf71ad9b7cb53d9ea4c81c0e